source/ddbus/exception.d

```diff
@@ -27,6 +27,21 @@ class DBusException : Exception {
   }
 }
 
+/++
+  Throw this from a handler to have it automatically translate to an Error
+  reply.
++/
+class DBusErrorReturn : Exception {
+  string errorName;
+
+  this(string _errorName, string errMsg, string file = __FILE__,
```

Comment:
this is ambiguous with `Exception.this(msg, file)`, which could lead to wrong use, especially with generic code.

Error name should just stay as public property with default ctor and be set after construction using some kind of builder method (like `.withErrorName`) or be constructed using a utility function like `throw dbusErrorReturn(name, msg, ...);`